lib/logstash/plugin_mixins/common.rb

```diff
@@ -0,0 +1,90 @@
+module LogStash
+  module PluginMixins
+    module SchemaRegistry
+      DEFAULT_SERIALIZER_CLASS = "org.apache.kafka.common.serialization.StringDeserializer"
+
+      def self.included(base)
+        base.extend(self)
+        base.setup_schema_registry_config
+      end
+
+      def setup_schema_registry_config
+        # Option to set key to access Schema Registry.
+        config :schema_registry_key, :validate => :string
+
+        # Option to set secret to access Schema Registry.
+        config :schema_registry_secret, :validate => :string
```

Comment:
```suggestion
        config :schema_registry_secret, :validate => :password
```

We should wrap secrets in the `LogStash::Util::Password` to keep them from escaping through logs etc. Note that this requires manual unwrapping with `LogStash::Util::Password#value` when we need to use it.